mobify/progressive-web-scaffold
# Selectorize checkout shipping

This PR converts access to the store for existing props to selectors, and parses more data from the desktop site for the checkout shipping page. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1111
 **Linked PRs**: #292 

## To Do
- [x] Selectorize access to the store for existing checkout shipping props
- [x] Get shipping method data for the page from magento's API
- [x] Removed some hardcoded content from the shipping address form
- [x] Parse available countries and regions from the desktop page 
- [ ] ~~Get initial values for form if user is logged in (no saved address)?~~

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart
- navigate to: https://www.merlinspotions.com/checkout/
- confirm the page works the same as it did before except now countries and regions are populated correctly
